core/domain/exp_domain.py

```diff
@@ -375,11 +375,11 @@ def validate(self):
             raise utils.ValidationError(
                 'Invalid audio filename: %s' % self.filename)
         extension = self.filename[dot_index + 1:]
-        if extension not in feconf.ACCEPTED_AUDIO_EXTENSIONS:
+        if extension not in feconf.ACCEPTED_AUDIO_EXTENSIONS.keys():
```

Comment:
Actually, better to omit keys() here. Using .keys() generates a new list and iterates over its elements, but you can do `if key not in dict` to do it more directly.